src/VisualStudio/Core/Def/ColorSchemes/Enhanced.xml

```diff
@@ -0,0 +1,810 @@
+            </Color>
+            <Color Name="xml literal - name">
+                <Foreground Type="CT_RAW" Source="FF863434" />
+            </Color>
+            <Color Name="xml literal - processing instruction">
+                <Foreground Type="CT_RAW" Source="FF5B5B5B" />
+            </Color>
+            <Color Name="xml literal - text">
+                <Foreground Type="CT_RAW" Source="FF3F3F3F" />
+            </Color>
+       </Category>
+    </Theme>
+    <Theme Name="Dark" GUID="{1ded0138-47ce-435e-84ef-9ec1f439b749}">
+        <Category Name="Roslyn Text Editor MEF Items" GUID="{75a05685-00a8-4ded-bae5-e7a50bfa929a}">
+            <Color Name="preprocessor text">
+                <Foreground Type="CT_RAW" Source="FFDCDCDC" />
+            </Color>
+            <Color Name="punctuation">
+                <Foreground Type="CT_RAW" Source="FFDCDCDC" />
+            </Color>
+            <Color Name="string - verbatim">
+                <Foreground Type="CT_RAW" Source="FFD69D85" />
+            </Color>
+            <Color Name="keyword - control">
+                <Foreground Type="CT_RAW" Source="FFD8A0DF" />
+            </Color>
+            <Color Name="operator - overloaded">
+                <Foreground Type="CT_RAW" Source="FFDCDCAA" />
+            </Color>
+            <Color Name="class name">
+                <Foreground Type="CT_RAW" Source="FF4EC9B0" />
+            </Color>
+            <Color Name="delegate name">
+                <Foreground Type="CT_RAW" Source="FF4EC9B0" />
+            </Color>
+            <Color Name="enum name">
+                <Foreground Type="CT_RAW" Source="FFB8D7A3" />
+            </Color>
+            <Color Name="interface name">
+                <Foreground Type="CT_RAW" Source="FFB8D7A3" />
+            </Color>
+            <Color Name="module name">
+                <Foreground Type="CT_RAW" Source="FF4EC9B0" />
+            </Color>
+            <Color Name="struct name">
+                <Foreground Type="CT_RAW" Source="FF86C691" />
+            </Color>
+            <Color Name="type parameter name">
+                <Foreground Type="CT_RAW" Source="FFB8D7A3" />
+            </Color>
+<!--
+            <Color Name="field name">
+            </Color>
+            <Color Name="enum member name">
+            </Color>
+            <Color Name="constant name">
+            </Color>
+-->
+            <Color Name="local name">
+                <Foreground Type="CT_RAW" Source="FF9CDCFE" />
+            </Color>
+            <Color Name="parameter name">
+                <Foreground Type="CT_RAW" Source="FF9CDCFE" />
+            </Color>
+            <Color Name="method name">
+                <Foreground Type="CT_RAW" Source="FFDCDCAA" />
+            </Color>
+            <Color Name="extension method name">
+                <Foreground Type="CT_RAW" Source="FFDCDCAA" />
+            </Color>
+<!--
+                <Color Name="property name">
```

Comment:
These classifications inherit their colors (based on the classification hierarchy). I wanted to leave all the classification names in the file in case colors are one day specified, but leaving it without a foreground entry causes the color compiler to error.